persistence/subsystem/offsetstore/transaction.go

```diff
@@ -0,0 +1,25 @@
+package offsetstore
+
+import (
+	"context"
+	"errors"
+)
+
+// ErrConflict is returned when the offset supplied as a parameter to method
+// Transaction.SaveOffset does not equal to the currently persisted offset.
```

Comment:
```suggestion
// ErrConflict is returned by transaction operations when a persisted
// offset can not be updated because the supplied "current" offset is
// out of date.
```